### PR TITLE
[FLINK-27513][flink-playgrounds] Update table walkthrough playground for 1.15

### PR DESCRIPTION
**Pull request by roy-michael**
1. Change Flink version to 1.5.2
2. Use Scala free dependencies
3. Add dependency required for tests to run

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Currently, the following playgrounds are available:
...
 ["Flink Operations Playground"](https://ci.apache.org/projects/flink/flink-docs-release-1.14/docs/try-flink/flink-operations-playground), which is part of the _Try Flink_ section of the Flink documentation.
 
-* The **Table Walkthrough** (in the `table-walkthrough` folder) shows to use the Table API to build an analytics pipeline that reads streaming data from Kafka and writes results to MySQL, along with a real-time dashboard in Grafana. The walkthrough is presented in detail in ["Real Time Reporting with the Table API"](https://ci.apache.org/projects/flink/flink-docs-release-1.14/docs/try-flink/table_api), which is part of the _Try Flink_ section of the Flink documentation.
+* The **Table Walkthrough** (in the `table-walkthrough` folder) shows to use the Table API to build an analytics pipeline that reads streaming data from Kafka and writes results to MySQL, along with a real-time dashboard in Grafana. The walkthrough is presented in detail in ["Real Time Reporting with the Table API"](https://ci.apache.org/projects/flink/flink-docs-release-1.15/docs/try-flink/table_api), which is part of the _Try Flink_ section of the Flink documentation.
```
Comment (alpinegizmo) [inline README.md]:
```suggestion
* The **Table Walkthrough** (in the `table-walkthrough` folder) shows how to use the Table API to build an analytics pipeline that reads streaming data from Kafka and writes results to MySQL, along with a real-time dashboard in Grafana. The walkthrough is presented in detail in ["Real Time Reporting with the Table API"](https://ci.apache.org/projects/flink/flink-docs-release-1.15/docs/try-flink/table_api), which is part of the _Try Flink_ section of the Flink documentation.
```

File: table-walkthrough/pom.xml [hunk 2]
```diff
@@ -72,9 +72,15 @@ under the License.
             <version>${flink.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+           <groupId>org.apache.flink</groupId>
+           <artifactId>flink-table-planner_${scala.binary.version}</artifactId>
+           <version>${flink.version}</version>
+           <scope>test</scope>
+       </dependency>
```
Comment (alpinegizmo) [inline table-walkthrough/pom.xml]: Try this instead -- I think it should work. We want to encourage using the scala-free versions wherever possible.

```suggestion
        <dependency>
           <groupId>org.apache.flink</groupId>
           <artifactId>flink-table-planner-loader</artifactId>
           <version>${flink.version}</version>
           <scope>provided</scope>
        </dependency>
        <dependency>
            <groupId>org.apache.flink</groupId>
            <artifactId>flink-table-runtime</artifactId>
            <version>${flink.version}</version>
            <scope>provided</scope>
        </dependency>
```

**Review by alpinegizmo — CHANGES_REQUESTED** [review]
Thanks for getting something working.

Let's see if we can update the dependencies w/o resorting to using `flink-table-planner_${scala.binary.version}`.

File: table-walkthrough/pom.xml (hunk 2 above)
Comment (roy-michael) [reply table-walkthrough/pom.xml]: this works as well. thanks for the clarification

**Comment by alpinegizmo** [comment]
Merged with 677a4bb892f6eba990617a8ecc699b14b10e7b6a.

**Pull request closed without merging**